libraries/botbuilder/tests/botFrameworkAdapter.test.js

```diff
@@ -1,6 +1,6 @@
 const assert = require('assert');
 const { TurnContext } = require('botbuilder-core');
-const { ChannelValidation } = require('botframework-connector');
+const connector = require('botframework-connector');
```

Comment:
Is this to make use of some static value? Combined with line [743](https://github.com/Microsoft/botbuilder-js/pull/772/files#diff-edbd5a55c0b905aed85dba82e657c9c9R743) where you take the `TokenApiClient` out of `connector`, I'm not sure why this seemingly odd syntax is being used.

Reply:
This is just to replace the TokenApiClient with my mock for testing. This takes advantage of the `module.exports` object being static and mutable. I can require in the whole thing, replace it with my mock and all other occurrences of `new TokenApiClient()` will be the mock.

Reply:
Ah I see what you mean; no, this looks great and seems cleaner than the alternative.